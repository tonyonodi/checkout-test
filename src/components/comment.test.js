import React from "react";
import renderer from "react-test-renderer";
import Comment from "./Comment";

it("renders comment correctly", () => {
  const tree = renderer
    .create(<Comment name="Bob" rating={2} comment="Not a huge fan" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders 0 star comment correctly", () => {
  const tree = renderer
    .create(<Comment name="Bob" rating={0} comment="Not a huge fan" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
