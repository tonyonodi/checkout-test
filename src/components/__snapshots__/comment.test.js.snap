// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 0 star comment correctly 1`] = `
<div
  className="sc-bdVaJa bDWFJH"
>
  <h3
    className="sc-bwzfXH kfvjYX"
  >
    Bob
  </h3>
  <div
    className="sc-htpNat gHoJWV"
  />
  <div
    className="sc-bxivhb bUZGRr"
  >
    <p>
      Not a huge fan
    </p>
  </div>
</div>
`;

exports[`renders comment correctly 1`] = `
<div
  className="sc-bdVaJa bDWFJH"
>
  <h3
    className="sc-bwzfXH kfvjYX"
  >
    Bob
  </h3>
  <div
    className="sc-htpNat gHoJWV"
  >
    <span>
      ⭐
    </span>
    <span>
      ⭐
    </span>
  </div>
  <div
    className="sc-bxivhb bUZGRr"
  >
    <p>
      Not a huge fan
    </p>
  </div>
</div>
`;
